### DISCUSSION TITLE
JinxFramer

### DISCUSSION THREAD

**Discussion by TelePortall** [vvip-68/GyverPanelWiFi#236] [Задавайте вопросы]
Здравствуйте! Подскажите, пожалуйста почему JinxFramer нормально воспроизводит и работает с вашими файлами .out отсюда https://drive.google.com/drive/folders/1Ldp4x8dMDcb0C3YqLD54SeyP2HF1yhGs.
Но когда я пытаюсь сам сформировать файл .out в Jinx и загрузить в программу JinxFramer, то лишь каждый 3ий кадр анимации оказывается верным, а остальные два кадра пишет ошибку "Размер матрицы не соответствует размеру кадра в файле". Решил попытался создать анимацию из каждого правильного, то есть каждого третьего кадра путем вырезания его и сохранения в виде набора картинок формата .р. Но в таком случае программа выдает ошибку "необрабатываемое исключение в программе" и прелагает продолжить работу игнорируя ошибку или закрыть программу. Однако ни в том, ни другом случае не удается сохранить картинку кадра в формате .р.
Может я неправильно распаковываю программу JinxFramer из вашего зипа? Просто создаю папку на рабочем диске и туда все файлы из папки JinxFramer в папке tools.
Не понимаю что я не так делаю, что не удается создать анимацию из Jinx
![image](https://user-images.githubusercontent.com/105998794/183264585-212c1d8e-da0b-4fd8-8e6e-ee081167a79f.png)

![image](https://user-images.githubusercontent.com/105998794/183264505-07a04294-52f6-4cc2-b1ef-1ae20d353535.png)

![image](https://user-images.githubusercontent.com/105998794/183264392-f05a071d-3cfb-4e3c-a791-17265f9ade9d.png)

**Comment by vvip-68** [comment]
Вы бы хоть файлики проблемные куда-то выложили. А так кто ж вам скажет что происходит.
Возможно неправильные настройки Jinx при создании и сохранении ролика.
Возможно на кадрах такого размера JinxFramer не тестировался  и они чем-то отличаются от кадров меньшего размера.
Проверка размера кадра - JinxFramer ищет определенные метки в файле, по которым он считает что это границы кадра. 
Возможно на таком размере метки-разделители кадра отличаются в соседних кадрах и JinxFramer не может определить начало/конец кадра и программу нужно подкорректировать...
